simpleSequential/test_unionTwoSet.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from unionTwoSet import unionTwoSet
+
+class TestUnionTwoSet(unittest.TestCase):
+    def test_equalSetIntegerInput(self):
+        self.assertEqual(unionTwoSet({2, 3}, {3, 2}), {2, 3})
+
+    def test_noneEqualSetIntegerInput(self):
```

Comment:
"non" or "none"? you are not consistence in different files. Sometimes you use "nonEqual", sometime you use "noneEqual".

I think you should stick to "non". Fix this.